cesium/internal/unary/db.go

```diff
@@ -28,9 +30,10 @@
 
 type DB struct {
 	Config
-	Domain     *domain.DB
-	Controller *controller.Controller[controlledWriter]
-	_idx       index.Index
+	Domain      *domain.DB
+	Controller  *controller.Controller[controlledWriter]
+	_idx        index.Index
+	openWriters *atomic.Int32Counter
```

Comment:
We're also tracking the number of open iterators here, so we should change the name to something more semantic.